pkg/resourcehandlers/github/github_resource_handler.go

```diff
@@ -272,35 +312,34 @@ func (gh *GitHub) URLToGitHubLocator(ctx context.Context, urlString string, reso
 		ghRL *ResourceLocator
 		err  error
 	)
-	// try cache first
-	if ghRL = gh.cache.Get(urlString); ghRL == nil {
-		if ghRL, err = parse(urlString); err != nil {
+	if ghRL, err = parse(urlString); err != nil {
+		return nil, err
+	}
+	if ghRL.Type == Wiki || len(ghRL.SHAAlias) == 0 {
+		return ghRL, nil
+	}
+	cachedRL := gh.cache.Get(ghRL)
+	if cachedRL != nil {
+		return cachedRL, nil
+	} else if resolveAPIUrl {
```

Comment:
go-ers don't like `else`. That should be the same:
```suggestion
	}
	if resolveAPIUrl {
```